docs/podman-build.1.md

```diff
@@ -4,7 +4,7 @@
 podman\-build - Build a container image using a Dockerfile.
 
 ## SYNOPSIS
-**podman** **build** [*options* [...]] **context**
+**podman build** [*options*] *context*
```

Comment:
I think this file is largely sourced from `buildah`, so making these changes upstream as well would be appreciated

Reply:
> I think this file is largely sourced from `buildah`, so making these changes upstream as well would be appreciated

Filed as projectatomic/buildah#839.